resources/assets/js/largo/components/annotationFilter.vue

```diff
@@ -109,11 +109,19 @@ export default {
         return {
             filterValues: {
                 Shape: availableShapes,
-                User: {}
+                User: {},
+                Confidence: {
+                    0: 'Very low (0 - 0.25)',
+                    1: 'Low (0.25 - 0.5)', 
+                    2: 'Medium (0.5 - 0.75)',
+                    3: 'High (0.75 - 0.9)',
+                    4: 'Very high (0.9 - 1.0)'
+                }
             },
```

Comment:
This could be a chance to add `>` and `<` operators to filter conditions. Instead of fixed intervals, users can enter one rule `>0.25` and another `<0.5` to get the same thing.